tests/algorithm/results/output.csv

```diff
@@ -0,0 +1,85 @@
+Test Case #,Expected Result,Match Result,Error
```

Comment:
Did you mean to commit this results file?

Reply:
no I didn't. I'll remove it from the pr